org.uqbar.project.wollok/src/org/uqbar/project/wollok/validation/WollokDslValidator.xtend

```diff
@@ -652,14 +686,6 @@ class WollokDslValidator extends AbstractConfigurableDslValidator {
 	}
 
 
-//  Confirm with @npasserini
```

Comment:
Didn't we already delete this?

Reply:
Maybe, but constructor-fixes was in another branch (developed afterwards, but previously merged)